code/game/objects/buckling.dm

```diff
@@ -5,6 +5,7 @@
 	var/buckle_lying = -1 //bed-like behavior, forces mob.lying = buckle_lying if != -1
 	var/buckle_require_restraints = 0 //require people to be handcuffed before being able to buckle. eg: pipes
 	var/mob/living/buckled_mob = null
+	var/has_buckled

```

Comment:
If someone is unbuckled but the buckled_mob var isn't updated then that particular thing is bugged. Adding a new var in case the other var breaks is very strange practice.